player/lua/ytdl_hook.lua

```diff
@@ -3,7 +3,7 @@ local msg = require 'mp.msg'
 
 local ytdl = {
     path = "youtube-dl",
-    minver = "2014.11.26",
+    minver = "2014.11.26", -- TODO: update to version with the new subtitles system?
```

Comment:
Videos can be played with the old version, but there won't be subtitles. I can keep it unchanged if you want.